templates/default/sqswatcher.cfg.erb

```diff
@@ -1,6 +1,7 @@
 [sqswatcher]
 region = <%= node['cfncluster']['cfn_region'] %>
 sqsqueue = <%= node['cfncluster']['cfn_sqs_queue'] %>
+healthsqsqueue = <%= node['cfncluster']['cfn_health_sqs_queue'] %>
```

Comment:
just because I'm picky :D we could rename this to `health_sqsqueue` just to use the same naming convention

Reply:
Done